File: templates/Uwp/_composition/MVVMBasic/Feature.IdentityForcedLogin/ViewModels/LogInViewModel_postaction.cs
namespace Param_RootNamespace.ViewModels
{
    public class LogInViewModel : Observable
    {
        public bool IsBusy
        {
            get => _isBusy;
            set
            {
                Set(ref _isBusy, value);
//{[{
                LoginCommand.OnCanExecuteChanged();
//}]}
            }
        }
    }
}

File: templates/Uwp/_comp/MVVMBasic/Feat.IdentityForcedLogin._VB/ViewModels/LogInViewModel_postaction.vb
Namespace ViewModels
    Public Class LogInViewModel

        Public Property IsBusy As Boolean
            Get
                Return _isBusy
            End Get
            Set(value As Boolean)
                [Set](_isBusy, value)
'{[{
                LoginCommand.OnCanExecuteChanged()
'}]}
            End Set
        End Property
    End Class
End Namespace
